src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -223,6 +223,16 @@ public class InstancesController : ControllerBase
             }
             catch (Exception partyLookupException)
             {
+                if (partyLookupException is ServiceException)
```

Comment:
We could have a separate catch for ServiceException.

Reply:
Yes, but that would also need to have the if.